test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -79,26 +84,28 @@ Describe "Test-Connection" -tags "CI" {
         }
 
         # In VSTS, address is 0.0.0.0
-        It "Force IPv4 with implicit PingOptions" -Skip:((Test-IsVstsLinux) -or (Test-IsVstsWindows)) {
-            $result = Test-Connection $realName -Count 1 -IPv4
+        It "Force IPv4 with implicit PingOptions" {
+            $result = Test-Connection $hostName -Count 1 -IPv4
 
             $result.Replies[0].Address              | Should -BeExactly $realAddress
-            $result.Replies[0].Options.Ttl          | Should -BeLessThan 128
+            $result.Replies[0].Options.Ttl          | Should -BeLessOrEqual 128
             if ($isWindows) {
                 $result.Replies[0].Options.DontFragment | Should -BeFalse
             }
         }
 
         # In VSTS, address is 0.0.0.0
-        It "Force IPv4 with explicit PingOptions" -Skip:((Test-IsVstsLinux) -or (Test-IsVstsWindows)) {
-            $result1 = Test-Connection $realName -Count 1 -IPv4 -MaxHops 10 -DontFragment
+        It "Force IPv4 with explicit PingOptions" {
+            $result1 = Test-Connection $hostName -Count 1 -IPv4 -MaxHops 10 -DontFragment
 
-            $result2 = Test-Connection $realName -Count 1 -IPv4 -MaxHops 1 -DontFragment
+            # explicitly go to google dns. this test will pass even in the destination is unreachable
+            # it's more about breaking out of the loop
+            $result2 = Test-Connection 8.8.8.8 -Count 1 -IPv4 -MaxHops 1 -DontFragment
 
             $result1.Replies[0].Address              | Should -BeExactly $realAddress
             # .Net Core (.Net Framework) returns Options based on default PingOptions() constructor (Ttl=128, DontFragment = false).
             # After .Net Core fix we should have 'DontFragment | Should -Be $true' here.
```

Comment:
Is this still valid? Or we can add this back?

Reply:
We have a tracking issue and an issue in CoreFX without any progress :-(